Wild fire reports (#14645)
demisto/content: Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/README.md

generates a WildFire PDF report.
This integration was integrated and tested with version xx of WildFire-Reports

## Configure WildFire Reports on Cortex XSOAR

1. Navigate to **Settings** > **Integrations** > **Servers & Services**.

> 2. Search for WildFire Reports.

Reviewer: Palo Alto Networks WildFire Reports

3. Click **Add instance** to create and configure a new integration instance.

    | **Parameter** | **Description** | **Required** |
    | --- | --- | --- |
    | Server base URL (e.g. https://192.168.0.1/publicapi) |  | True |
    | API Key |  | True |
    | Source Reliability | Reliability of the source providing the intelligence data. | True |
    | Trust any certificate (not secure) | Trust any certificate \(not secure\). | False |
    | Use system proxy settings | Use system proxy settings. | False |
    | Return warning entry for unsupported file types |  | False |

4. Click **Test** to validate the URLs, token, and connection.
## Commands
You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
After you successfully execute a command, a DBot message appears in the War Room with the command details.
### wildfire-report
***
Retrieves results for a file hash using WildFire.

#### Base Command

`wildfire-report`
#### Input

| **Argument Name** | **Description** | **Required** |
| --- | --- | --- |
| md5 | MD5 hash to check. | Optional | 

> | sha256 | SHA256 hash to check. | Optional | 

Reviewer: i think it's better to put sha256 first (and after that show md5) because that will probably be the more important and common command

| hash | Deprecated - Use the sha256 argument instead. | Optional | 
| verbose | Receive extended information from WildFire. Possible values are: true, false. Default is false. | Optional | 

> #### Context Output

Reviewer: not sure this is relevant for this integration, since it's only a PDF download

| WildFire.Report.Network.UDP.Port | string | Submission related ports, in UDP protocol. | 
| WildFire.Report.Network.TCP.IP | string | Submission related IPs, in TCP protocol. | 
| WildFire.Report.Network.TCP.Port | string | Submission related ports, in TCP protocol. | 
| WildFire.Report.Network.DNS.Query | string | Submission DNS queries. | 
| WildFire.Report.Network.DNS.Response | string | Submission DNS responses. | 
| WildFire.Report.Evidence.md5 | string | Submission evidence MD5 hash. | 
| WildFire.Report.Evidence.Text | string | Submission evidence text. | 
| WildFire.Report.detection_reasons.description | string | Reason for the detection verdict. | 
| WildFire.Report.detection_reasons.name | string | Name of the detection. | 
| WildFire.Report.detection_reasons.type | string | Type of the detection. | 
| WildFire.Report.detection_reasons.verdict | string | Verdict of the detection. | 
| WildFire.Report.detection_reasons.artifacts | unknown | Artifacts of the detection reasons. | 
| WildFire.Report.iocs | unknown | Associated IOCs. | 
| WildFire.Report.verdict | string | The verdict of the report. | 

#### Command Example

> ```!wildfire-report hash=abcdef1234567890abcdef1234567890abcdef1234567890abcdef1234567890```

Reviewer: you can't say this is a command example, and in the docs before say that this is a deprecated attribute
please change to sha256 instead of hash

> #### Context Example

Reviewer: is this really what returns from this command?
i don't understand how the PDF returns this..

Generates a Palo Alto Networks WildFire PDF report.

> This integration was created and tested with version 10.1 of PAN-OS

Reviewer 2: ```suggestion
This integration was created and tested with version 10.1 of WildFire.
```

Generates a Palo Alto Networks WildFire PDF report.

> This integration was created and tested with version 10.1 of WildFire

Reviewer 2: ```suggestion
This integration was created and tested with version 10.1 of WildFire.
```

## Configure Palo Alto Networks WildFire Reports on Cortex XSOAR

1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
2. Search for Palo Alto Networks WildFire Reports.
3. Click **Add instance** to create and configure a new integration instance.

    | **Parameter** | **Description** | **Required** |
    | --- | --- | --- |
    | Server base URL (e.g., https://192.168.0.1/publicapi) |  | True |

>     | API Key |  | True |

Reviewer 2: ```suggestion
    | API Key |  | False |
```
